tests/test_minimal/test_tools/test_backend_and_dataset_configuration/test_helpers/test_configure_backend_overrides.py

```diff
@@ -0,0 +1,77 @@
+"""Unit tests for `get_default_dataset_configurations`."""
+from pathlib import Path
+from typing import Literal
+
+import numcodecs
```

Comment:
Question: should this be added to `requirements_testing`? how is it making it here?

Reply:
OK. This makes sense.